torch/nn/quantized/functional.py

```diff
@@ -452,6 +452,25 @@ def hardswish(input, scale, zero_point):
         raise ValueError("Input to 'quantized.hardswish' must be quantized!")
     return torch._ops.ops.quantized.hardswish(input, scale, zero_point)
 
+def threshold(input, threshold, value):
+    # type: (Tensor, float, float) -> Tensor
+    r"""Applies the quantized version of the threshold function element-wise:
+
+    .. math::
+        \text{threshold}(x) = x \text{ if } x > \text{threshold else } \text{ value}
```

Comment:
A better math equation would be

```latex
x = \begin{cases}
  x & \text{if~} x > \text{threshold} \\
  \text{value} & \text{otherwise}
\end{cases}
```

that will show 

![image](https://user-images.githubusercontent.com/4216323/83477429-13805700-a448-11ea-8f4d-a5dd53356a5d.png)

Reply:
Sounds good!